Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -112,6 +116,14 @@ private void OnInteractUsingEvent(EntityUid uid, FlammableComponent flammable, I
             if (!isHotEvent.IsHot)
                 return;
 
+            // Only apply stacks to entities tagged with specific materials
+            // Otherwise  mobs would too easily be able to set each other on fire
+            // In future, support Wood, Fuel, Cloth, etc.
+            if (_tags.HasTag(uid, "Paper"))
```

Comment:
Using tags here is a bad idea. I'd prefer if you used a property on the flammable component itself (and even then, I'm not too fond about that...)